lib/authorizer/jwt.js

```diff
@@ -0,0 +1,221 @@
+const _ = require('lodash'),
+    jwt = require('jsonwebtoken'),
+
+    // jwt key constants
+    BEARER_AUTH_PREFIX = 'Bearer',
+    ALGORITHM = 'algorithm',
+    HEADER = 'header',
+    HEADER_ALGORITHM = 'alg',
+    PAYLOAD = 'payload',
+    SECRET = 'secret',
+    IS_SECRET_BASE_64_ENCODED = 'isSecretBase64Encoded',
+    PRIVATE_KEY = 'privateKey',
+    PASSPHRASE = 'passphrase',
```

Comment:
What are the differences between `secret`, `privateKey` and `passphrase`? Should we document these?

Reply:
Yes @coditva documented in the schema comments block